README.md

```diff
@@ -1,28 +1,56 @@
 # Zip_Backfill
 
-The primary file is Zip_Backfill.py, all other files are to test its accuracy.
+**Description**: `Zip_Backfill.py` takes an OpenAddresses files and, for rows that do not have zip codes, compares the row's lat and lon to a shape file of zip codes in order to assign it a zip code. The code is designed to work with a Census ZCTA `.shp` file.
 
-Zip_Backfill runs from the command line and takes as input two files: input and output. Input is a .csv file with addresses that need to be backfilled. This file should be in the format: Lon, Lat, Number, Street, City, District, Region, Zip, ID. Output is a .csv file that the backfilled addresses will be written into. Address rows will be returned in the same format as they are inputted.
+All other files are to test the efficacy of `Zip_Backfill.py`.
 
-##Dependencies
+##Usage
 
-###All
+- To install dependencies run `pip install -r requirements.txt`
 
-- Python 2.7
+- Download the 2015 TIGER ZCTA `zip` [here](ftp://ftp2.census.gov/geo/tiger/TIGER2015/ZCTA5/)
```

Comment:
If you look [here](https://github.com/cfpb/zip-backfill/blob/kgudel/README.md) it looks like Github markdown doesn't like the `ftp` link. Is there a census page that we can link to that has this link?